rubocop.yml

```diff
@@ -50,6 +49,12 @@ Style/AccessorMethodName:
   Enabled: false
 Style/Alias:
   Enabled: false
+Style/BlockDelimiters:
+  EnforcedStyle: semantic
```

Comment:
From my POV `line_count_based` is better, because inline `do end` looks clumsy. WDYT?